sa/database.go

```diff
@@ -182,3 +183,10 @@ func initTables(dbMap *gorp.DbMap) {
 	dbMap.AddTableWithName(core.SignedCertificateTimestamp{}, "sctReceipts").SetKeys(true, "ID").SetVersionCol("LockCol")
 	dbMap.AddTableWithName(core.FQDNSet{}, "fqdnSets").SetKeys(true, "ID")
 }
+
+func safeSelectOne(m *gorp.DbMap, holder interface{}, q string, args ...interface{}) error {
+	if strings.ContainsAny(q, "'\"\n\r\\") {
+		return errors.New("Query contains unsafe character")
```

Comment:
I had a stab at implementing this but it ends up either seeming slightly confusing (i.e. `Query contains an unsafe character ("'\"\n\r\\\x00")`, which one was it? was it all of them? why are these ones encoded and hence safe? etc) or complicated to keep in sync (i.e. using a different string to indicate what is in the set than the actual string of unsafe characters).